.github/workflows/pr-lint.yml

```diff
@@ -0,0 +1,16 @@
+name: PR Quality Check
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, edited]
+jobs:
+  check-pr-name:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check ticket name conforms to requirements
+        run: 'echo ${{ github.event.pull_request.title }} | grep -i -E -q "^((apm|niad)-[0-9]+): ([a-zA-Z]+)|(dependabot\/)"'
```

Comment:
If we are using the -i flag, don't think that A-Z is necessary in "[a-zA-Z]" part